pyproject.toml

```diff
@@ -191,7 +191,13 @@ sdist.include = ["fast_pauli/__version__.py"]
 write_to = "fast_pauli/__version__.py"
 
 [tool.cibuildwheel]
-build = ["cp310-manylinux_x86_64", "cp311-manylinux_x86_64", "cp312-manylinux_x86_64", "cp313-manylinux_x86_64"]
-build-frontend = "build"
+skip = "pp*"
```

Comment:
what does this `skip` ?

Reply:
pp* filters out all PyPy builds

Reply:
For context @stand-by, those builds require extra configuration and we don't think it's worth the effort rn since most users will be working with cpython